### PR TITLE
add Yukon shields

### PR DESCRIPTION
**Pull request by claysmalley**
Added shields for Yukon territory. This adds another special case, as shields for different routes have different colors:

* Yellow: routes 2, 3
* Blue: routes 5, 11
* Green: route 6
* Red: all other routes

Screenshot from 2022-03-24 09-47-33
Screenshot from 2022-03-24 09-48-07
Screenshot from 2022-03-24 09-48-33

### CODE REVIEW

File: style/js/shield_defs.js [hunk 1]
```diff
@@ -1251,6 +1253,19 @@ export function shieldLighten(network, ref) {
...
+        case "2":
+        case "3":
+          return "#ce9d00";
+        case "5":
+        case "11":
+          return "#0054a8";
+        case "6":
+          return "#006a4d";
+        default:
+          return "#bf2033";
```
[the remark this answers is not in the document]
Comment (claysmalley) [reply style/js/shield_defs.js:1267]: I made those Wikimedia SVGs :wink: I eyeballed the colors as much as I could when I made them; they're not authoritative.

Here's what it looks like with the Wikimedia SVG colors:

Screenshot from 2022-03-24 11-11-08
Screenshot from 2022-03-24 11-11-34

**Comment by ZeLonewolf** [comment]
This shield situation is so bizarre that I'm dying to know the back-story...

**Comment by claysmalley** [comment]
My hunch is that it's something similar to [Florida's previous system](https://www.aaroads.com/florida/florida-us-highways/) of signing US routes, assigning a color to each route as another navigational aid. Florida eventually phased it out in favor of federal highway money, but Canada doesn't have the same constraints.

File: style/js/shield_defs.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (1ec5) [reply style/js/shield_defs.js]: Do we need to use literally the same colors as the signs, or can we take some liberties? These shields would be some of the first that don’t use the color scheme in CONTRIBUTING.md, even considering non-MUTCD countries.

[the remark this answers is not in the document]
Comment (claysmalley) [reply style/js/shield_defs.js]: Going by the existing color scheme:

Screenshot from 2022-03-24 21-57-46
Screenshot from 2022-03-24 21-58-15

Red, blue and green are a reasonable match. But yellow is illegible.

I think I'll stick with the previous yellow, and change the rest of the colors to match the color scheme:

Screenshot from 2022-03-24 22-06-12

**Comment by ZeLonewolf** [comment]
Turns out they're unreadable in real life, too...
image

**Review by ZeLonewolf — APPROVED** [review]
These look good enough to me.  I don't think we ever anticipated yellow borders on shields when we came up with the color palette but here we are.  Is this ready to merge or are we still playing with the colors?

**Comment by claysmalley** [comment]
I think the colors are as good as they'll get. Go ahead and merge.

**Pull request merged**